pkg/sloop/webfiles/index.html

```diff
@@ -104,9 +104,13 @@ <h2>Links</h2>
             integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
             crossorigin="anonymous"></script>
 
+    <!-- This is the list of scripts needed by resource.html, because inner html can not use includes -->
     <script src="https://unpkg.com/axios/dist/axios.min.js" async="" type="text/javascript"></script>
     <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js" async="" type="text/javascript"></script>
     <script src="https://unpkg.com/nprogress@0.2.0/nprogress.js" async="" type="text/javascript"></script>
+    <script src="https://cemerick.github.io/jsdifflib/diffview.js" type="text/javascript" ></script>
+    <script src="https://cemerick.github.io/jsdifflib/difflib.js" type="text/javascript" ></script>
```

Comment:
nit: let's add `async` attribute as well.

Reply:
Im worried that I dont know how to test this if it introduces a race condition.  Let's save this for another PR.  But good suggestion!